scss/_navbar.scss

```diff
@@ -108,7 +108,7 @@
   }
 
   .dropdown-menu {
-    position: static;
+    position: absolute;
```

Comment:
Have you tried to remove entirely the `.dropdown-menu` rule here? We might have this absolute position by default (haven't tried).

Reply:
I gave this a shot and looks like you are correct, it appears to behave the exact same when removing the rule entirely. I've updated this branch accordingly with a new commit.

Reply:
I don't quite see what should be changed about that documentation header, from what I've seen it still looks the same in the new version as in the old version?

Reply:
> I don't quite see what should be changed about that documentation header, from what I've seen it still looks the same in the new version as in the old version?

Reference at https://getbootstrap.com:

![Screenshot 2024-12-31 at 08 23 37](https://github.com/user-attachments/assets/856a0187-d741-419f-9e01-b14e631edb07)

In this branch, the dropdown menu is too close to the left edge.

![Screenshot 2024-12-31 at 08 23 46](https://github.com/user-attachments/assets/b242cdd3-5aa6-44d7-bdcc-c0b7c8c36d49)